packages/hierarchy-browser/src/CommunityCard/CommunityCard.tsx

```diff
@@ -80,13 +62,17 @@ export const CommunityCard: React.FC<ICommunityCardProps> = memo(
 			isOpen,
 			toggleOpen,
 			filterProps,
-		] = useCommunityData(dataProvider, isOpenProp, maxLevel, size)
+		] = useCommunityData(dataProvider, isOpenProp, maxLevel)
 
 		const [
 			adjacentCommunities,
 			isAdjacentEntitiesLoading,
 		] = useAdjacentCommunityData(dataProvider, isOpen, neighborsLoaded)
 
+		const size = useMemo(() => dataProvider.size, [dataProvider.size])
```

Comment:
This isn't worth memoizing, since it's not saving any compute. We compute dataProvider.size anyway in the dependency list.